proto/elys/clob/query.proto

```diff
@@ -152,4 +158,13 @@ message AllPerpetualADLRequest {
 message AllPerpetualADLResponse {
   repeated PerpetualADL adl_list = 1 [ (gogoproto.nullable) = false ];
   cosmos.base.query.v1beta1.PageResponse pagination = 2;
+}
+
+message AllPerpetualsWithLiquidationPriceRequest {
+  cosmos.base.query.v1beta1.PageRequest pagination = 1;
```

Comment:
you can market_id as well, if it;s 0 then it sends all perpetual (current implementation) and if not 0 then it's send perpetual of thar market only